docs/Tutorials/Advanced/ticketing/introduction.md

```diff
@@ -2,26 +2,148 @@
 * No wallet is needed to enter the event or receive a POAP.
 * No wifi is needed at the door.
-* An NFT is minted on-demand for each user that attends the event.
 * Users can optionally onboard onto NEAR if they don't have a wallet.
+* Users that choose to onboard can receive a Proof-of-Attendence NFT, although this is optional.
+
+These features open the door to an entirely new class of non-technical attendees and create a smoother experience compared to existing NFT ticketing solutions.
+
+In this tutorial, you'll be creating a ticketing system with those features, and incorporating a POAP to further reward your community members that come out to your events. 
+
+:::info note
+The NFT POAP is optional to include as the event organizer. You may omit it, or replace it with your own function call if you wish. In this tutorial, the POAP will be minted on the second key use. 
+:::
+
+---
+
+## Prerequisites
+
+For the advanced tutorials, you can choose to run the scripts on your own machine. To do so, you must have the following:
+
+1. [Node JS](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm)  
+2. [NEAR-API-JS](https://docs.near.org/tools/near-api-js/quick-reference#install)  
+3. [Keypom JS SDK](https://github.com/keypom/keypom-js#installation)
+
+If you want to reference the finished code, that can be found [here](https://github.com/keypom/keypom-js/tree/min/ticketing-tutorial/docs-advanced-tutorials/ticket-app). To follow along and build out this ticketing app, see the steps below. 
+
+---
+
+## Creating your Project
+In this section, you're going to create your project and install the SDK to prepare for the tutorial. If you have a completed script and have installed the SDK, you can skip [forward](architecture.md).
+
+First, navigate to your desired home folder and use the `create-near-app` command to create a new project.
+``` bash
+npx create-near-app my-ticket-app
+```
+:::note
+You may realize that there is a `contract` and `frontend` folder. The `create-near-app` package allows you to build out a NEAR smart contract and quickly hook it up with a frontend. This tutorial will not involve writting any smart contracts, so the `contract` folder can be ignored. 
+:::
+
+You'll then need to create a `package.json` inside the `frontend` folder. You can accept all default values that NPM suggests.
+
+``` bash
+cd frontend && npm init
+```
+
+Then, you can install the following libraries, including the Keypom JS SDK.
+
+``` bash
+npm install keypom-js && npm install react && npm install react-dom && npm install react-router-dom &&
+npm install qrcode.react && npm install react-zxing && npm install web-vitals
+```
+
+Next, you'll want to navigate to `my-ticket-app/package.json` and make the following replacement:
+
+```json
+"scripts": {
```

Comment:
This is not intuitive to the reader - let's just copy the whole scripts section of the package.json and tell them to replace their existing script